SDK/AppCenter/Microsoft.AppCenter.Android/Microsoft.AppCenter.Android.csproj

```diff
@@ -15,7 +15,7 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
-    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
```

Comment:
We can't use that we pinned mono version to 5.4 as it triggers incompatibilities in the builds, not that we were not using 8.1 either.